package.json

```diff
@@ -30,6 +30,7 @@
     "@changesets/pre": "^1.0.4",
     "@changesets/read": "^0.4.6",
     "@manypkg/get-packages": "^1.1.1",
+    "@octokit/webhooks-definitions": "^3.67.3",
```

Comment:
It seems that @octokit/webhooks-definitions is deprecated, right?
https://www.npmjs.com/package/@octokit/webhooks-definitions

![Captura de pantalla 2022-01-20 a las 12 12 18](https://user-images.githubusercontent.com/26763673/150328117-febe062a-e97f-4d4f-908a-bae7e175fbb9.png)

[They recommend here](https://github.com/octokit/webhooks/issues/447#issue-862139322) using one of the following 3 packages now, depending on what you need:

```
@octokit/webhooks-types - TypeScript definitions
@octokit/webhooks-schemas - JSON Schemas
@octokit/webhooks-examples - Examples for all webhook events
```

Reply:
removed it caballero